### DISCUSSION TITLE
New icon for MacOS app

### DISCUSSION THREAD

**Discussion by wavetape** [ppy/osu#17557] [Ideas] [answered]
**fix app icon on macos, it is out of common macos design**

**logo (now):**
<img width="136" alt="image" src="https://user-images.githubusercontent.com/55458918/160851431-dcfcdb26-7567-4daf-b453-eeab5f8e8ac7.png">
**logo (my version) :**
<img width="130" alt="image" src="https://user-images.githubusercontent.com/55458918/160851683-79c302f6-5592-4e09-99dd-0eb2b7ada21d.png">

<img width="130" alt="image" src="https://user-images.githubusercontent.com/55458918/160852465-50496eb5-65c0-44c2-a0c0-9761a488bf11.png">

**Comment by peppy** [comment, marked as answer]
macOS changes the system style of icons every other year. we won't be matching it for the time being. many other apps also don't.

<img width="214" alt="Safari 2022-03-30 at 14 26 30@2x" src="https://user-images.githubusercontent.com/191335/160858611-002a692c-fc6b-4dc5-8d54-7475413bac64.png">
